openupgrade_scripts/scripts/account/14.0.1.1/pre-migration.py

```diff
@@ -416,6 +416,20 @@ def fill_account_payment_partner_id(env):
     )
 
 
+def delete_xmlid_existing_groups(env):
+    env.cr.execute(
```

Comment:
Why removing this? You are breaking possible import templates.

Reply:
I think it makes sense because of the auto generation logic that modules do not add xml records for this. Are there examples of such XML records?

Reply:
OK, nevermind. I was thinking about importing accounts, but the group is now auto-computed, so now need to reference it from the import.